lib/controllers/object.js

```diff
@@ -225,6 +226,62 @@ exports.getObjectAcl = async function getObjectAcl(ctx) {
   };
 };
 
+
+/**
+ * POST Object
+ * Post an object into a bucket
```

Comment:
This could use a better description - the first sentence in the link for S3's API docs usually provides a good summary if you're feeling lazy.

Reply:
done